banano-quest/banano-quest/ViewControllers/CompleteQuestViewController.swift

```diff
@@ -63,17 +63,12 @@ class CompleteQuestViewController: UIViewController, CLLocationManagerDelegate {
     func refreshView() {
         // Details view
         bananoCountLabel.text = "0/\(quest?.maxWinners ?? 1)"
-        prizeValueLabel.text = "\(quest?.prize ?? 1).0 USD"
+        prizeValueLabel.text = "\(quest?.prize ?? 1) ETH"
```

Comment:
You need to divide the prize by the `maxWinners properties`:

`"\(quest?.prize/quest?.maxWinners ?? 1) ETH"`